#wall 1 ax+by+c=0
float64 a1
float64 b1
float64 c1
#wall 2 ax+by+c=0
float64 a2
float64 b2
float64 c2
#corner
float64 cx
float64 cy
float64 reliability
#edge
float64 e1x
float64 e1y
float64 e2x
float64 e2y